cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/scorecompiler/ScoreCompilerImpl.java

```diff
@@ -142,11 +142,12 @@ private ExecutionPlan compileToExecutionPlan(Executable executable) {
 		Collection<Input> result = new ArrayList<>();
 		for(Executable executable : executables) {
 			result.addAll(getSystemProperties(executable.getInputs()));
-			if(executable instanceof Flow) {
-				for(Task task : ((Flow)executable).getWorkflow().getTasks()) {
-					result.addAll(getSystemProperties(task.getInputs()));
-				}
-			}
+            // TODO - task args - don't support sys prop for task args
```

Comment:
correct - I am pushing now some changes regarding the compiler (enhancements, test updates)